### PR TITLE
Build on Gaea

### PR DESCRIPTION
**Pull request by DavidBurrows-NCO**
# Description
Add/update module file scripts for compiling gfs_utils on Gaea-C5

  Refs #66

# Type of change
- New feature (adds functionality)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?
- Clone and build on Gaea-C5

# Checklist
- [ ] Any dependent changes have been merged and published
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: modulefiles/gfsutils_common.lua [hunk 1]
```diff
@@ -5,7 +5,7 @@ Load common modules to build GFS utilities on all machines
 local netcdf_c_ver=os.getenv("netcdf_c_ver") or "4.9.2"
 local netcdf_fortran_ver=os.getenv("netcdf_fortran_ver") or "4.6.1"
 
-local bufr_ver=os.getenv("bufr_ver") or "11.7.0"
+local bufr_ver=os.getenv("bufr_ver") or "12.0.1"
```
Comment (aerorahul) [inline modulefiles/gfsutils_common.lua]: This is the only thing that makes me pause.
Do we have `bufr_ver=12.0.1` installed as part of spack-stack 1.6.0 on other RDHPCS machines that EPIC maintains?  If not, then this change will break building of gfs-utils on Hera, Orion and Hercules.
I will check on WCOSS2, but the move to bufr v12 from v11 needs to be coordinated with the DA software as well.

**Review by aerorahul — CHANGES_REQUESTED** [review]
I only have 1 comment that holds this PR back.
It is the same comment I had in #64

**Review by aerorahul — APPROVED** [review]
Looks good.
Thanks for adding the capability to build on Gaea.

File: modulefiles/gfsutils_common.lua (hunk 1 above)
Comment (DavidBurrows-NCO) [reply modulefiles/gfsutils_common.lua]: Morning! I reverted the bufr load to v11. And I see you approved. Thanks!

**Comment by DavidBurrows-NCO** [comment]
@aerorahul Morning..The epic software team got back to me yesterday and had a small correction to the gaea module file for gfs_utils..It doesn't change anything but eliminates a python version number warning message. I have committed that change and rebuilt. What's the process to correct this PR? New PR for bug fix? Reopen here? Thanks

**Comment by aerorahul** [comment]
> @aerorahul Morning..The epic software team got back to me yesterday and had a small correction to the gaea module file for gfs_utils..It doesn't change anything but eliminates a python version number warning message. I have committed that change and rebuilt. What's the process to correct this PR? New PR for bug fix? Reopen here? Thanks

Yes.  A new PR with a hotfix/bugfix is generally how this is handled.

**Comment by DavidBurrows-NCO** [comment]
Great. Will do. Thanks.

**Pull request merged**